.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
       env:
         - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
         - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
-        - SPARK_VERSION=2.3.4
+        - SPARK_VERSION=2.4.4
```

Comment:
Let's do this after Spark 3.0 is released. Officially we require Spark 2.4+ but it's better to make it working Spark 2.3 too for now.

Reply:
okay i got it :)